lib/fastlane/plugin/firebase_app_distribution/client/aab_info.rb

```diff
@@ -0,0 +1,42 @@
+class AabInfo
+  # AAB states
+  class AabState
+    UNSPECIFIED = "AAB_STATE_UNSPECIFIED"
```

Comment:
Small nit, since these strings don't have any variable interpolation, you can use single quotes instead of doubles.  This style guideline might be implemented haphazardly in the surrounding code, but I'll typically make this adjustment when changing or moving lines to have the code converge on this convention.

Reply:
Agreed, double quoted made me nauseous but was following the surrounding code :) will update.

Reply:
Done.